### PR TITLE
feat: update and add new credit card icons

### PR DESCRIPTION
**Pull request by starsirius**
https://artsyproduct.atlassian.net/browse/EMI-2063

We want to take this opportunity to standardize our credit card icons across the board and add a few new ones, such as Carte Bancaire, which is required for the [co-branded cards compliance](https://artsyproduct.atlassian.net/browse/EMI-2002) project.

Here is the [link to Figma](https://www.figma.com/design/gZNkyqLT8AU3T61tluVJyB/Artsy-3.1-Design-System?node-id=10968-22354&node-type=frame&t=ce3O7DF5JpHzAGEY-11) and attaching an image below too:

Screenshot 2024-10-09 at 4 59 07 PM

### CODE REVIEW

File: src/Amex.svg
Comment (starsirius) [inline src/Amex.svg:1]: Oh noticing the h/w differences. Is that the standardized dimension I should use?

**Comment by starsirius** [comment]
Realize @dzucconi is out this week. Tagging you @damassi for feedback/approval! 🙏

File: src/Amex.svg
Comment (damassi) [reply src/Amex.svg:1]: 18 is the typical dimension -- i think its fine!

**Review by damassi — APPROVED** [review]

File: src/CarteBancaire.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
...dHPLnacdPPIWnI6eeXO046OeXO04QWpNZtTXiazqsaxqprLGs1EzaxrFZSYtRM2ooO0Ws6qKMavxlKDrDjGqwoxqpsY1RKD1UcYtYUHqsIeqwoxqspyD1WHB6rCg9VFB6onIPVZTg9VHBaolIPVZTg9Vhweqw4LVYUHqsOD1WOkgtVhQWqw5B6onBbrDgt1lOQeqw5BbrDgtVhwWqJyC1WOkFqsKQW6x0kFqicgt1hwOqw5Baoukgt1hwWqLpGUkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkPMe6Pk+lzGjS4jRpsxo02YoNNiNGlxGjT88qi6eeQtOR0c8hacdPPLnacjp55C046OeXO046eeQtKR0c8udpx088hacjo55c7Tjp55CnI6eWXO046OeQtOR08sudpx0c8hacjp5Zc7Sjp55C10jo55c7SkdPPIWukdPPLnaUhpPIFN6xseVJmprypsYrGs1EzaxsZtYUYtTYzawmLUUY1WUmLWNYtRRjVE4xqooPVYTFZSg9VhRjVVKMaolB6qKDtYcHqsOD1WUoPVYUY1WU4PVYcg9VhwWqylB6rDkHqsOD1RpwWqwpBarDg9VhyC1WGPVYUgtVjpIPVE5BaqOC1ROQeqw4LdYcFqsOQWqw5BbonBarDkFqsdJBarDgdUTkFqsOQWqyukFujTkGwkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkknmI90j5OLmNGmxGjS4jRpsRo02Y0KbEaLo55G0pHRzy52nHTzyFKR08sudrpHRzyFpSOnllztOOnnkLTjo5Zc7Tjp55C046eeXO05HRzyFpx088udpSOjnkLXSOnnlztKR0c8hacdPPLnacjp55C046OeXO05HTzy52nHRiBacbFqTWbU2PE1i1jWdVNjNY1msJi1EzaytjFqKMaqJii2MaqKMarDjFrCjFrCHqooxqiUHqooxawoPVYUHqsOMarKUHqsOD1WHB6qKD1ROQeqynB6rCg9VhyD1WU5B6rDgtVhQWqw5B6rKcHqsOC1RKC1WHIPVZXSQWqw4PdZSkFqsdJBaonBarCkFusdJBarKcFqsOQW6JyC1WOkgt1lOB1ROQeqx0kDusOQVvtE48SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSf/Z"/>
```
Comment (joeyAghion) [inline src/CarteBancaire.svg:15]: This is an extremely large blob to be downloading everywhere (and, now, even preloading ahead of page content). There must be a smaller way to encode a similar gradient?

Even better: avoid loading this icon and others like it until necessary.

Comment (dzucconi) [reply src/CarteBancaire.svg:15]: It should only be downloaded individually. This package is set up to isolate imports.

Comment (joeyAghion) [reply src/CarteBancaire.svg:15]: This was surfaced by the Chrome coverage tool ([discussed here](https://artsy.slack.com/archives/C05EEBNEF71/p1731018412391149?thread_ts=1731017791.751689&cid=C05EEBNEF71)). If I'm interpreting the results correctly, it seemed to be downloading on the production home page where but was unused.

Comment (starsirius) [reply src/CarteBancaire.svg:15]: I exported from the [Figma design](https://www.figma.com/design/gZNkyqLT8AU3T61tluVJyB/Artsy-3.1-Design-System?node-id=10968-22354&node-type=frame&t=gf1otpEFIofDjmag-0). Guess it's for the gradient background? Maybe there is a way to optimize?

Screenshot 2024-11-08 at 5 11 37 PM

Comment (dzucconi) [reply src/CarteBancaire.svg:15]: Yea, design needs to redraw that to not use an image

**Pull request merged**